docs/blog_post.md

```diff
@@ -0,0 +1,56 @@
+DRAFT ONLY--DO NOT QUOTE
+
+# ShopRunner Data Science for Good: Automating Urban Wildlife Classification
+
+![coyote](https://github.com/uptake/autofocus/raw/master/gallery/coyote1.jpg) 
+<!--TK image caption formatting-->
+Software developed by data scientists at ShopRunner and Uptake supports the work of urban wildlife researchers by automatically classifying images taken by motion-activated cameras, such as this image of a coyote in the Chicago area.
+
+The growth of cities threatens animal species that do not adapt well to urban environments. **The Lincoln Park Zoo's Urban Wildlife Institute studies how urbanization affects wildlife in part by collecting images from motion-activated cameras**, beginning in 2010 around Chicago but now from more than twenty urban areas. For almost a decade, volunteers have been helping in their efforts by identifying the animal species that appear in each of those images in the online [Zooniverse](https://www.zooniverse.org/projects/zooniverse/chicago-wildlife-watch) crowdsourcing platform. However, the rate of image collection now exceeds the rate of image labeling. The information that the images contain is effectively unavailable to researchers until they have been labeled, so **image labeling has become a bottleneck that holds back their research.**
+
+Fortunately, it is now possible to create computer programs that label images with accuracy comparable to that of humans. Moreover, ShopRunner's data science team has experience building such programs to classify products across ShopRunner's retail network for its marketplace app District. For over two years, **data scientists from ShopRunner and Uptake have been working with the Urban Wildlife Institute to use automated image classification techniques to unblock the institute's research efforts.** 
```

Comment:
This classification work isn't just for District, so I'd say "classify products across ShopRunner's retail network, in particular for its marketplace app District" (or something like that).